opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -165,13 +165,27 @@ message LogRecord {
 
   // A unique identifier for a trace. All logs from the same trace share
   // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
-  // is considered invalid. Can be set for logs that are part of request processing
-  // and have an assigned trace id. [Optional].
+  // is considered invalid.
+  //
+  // This field is optional.
+  //
+  // The receivers SHOULD assume that the log record is not associated with a
+  // trace if any of the following is true:
+  //   - the field is not present,
+  //   - the field is equal to zero-length bytes sequence (or to empty string in OTLP/JSON),
+  //   - the field contains an invalid value.
   bytes trace_id = 9;
 
   // A unique identifier for a span within a trace, assigned when the span
   // is created. The ID is an 8-byte array. An ID with all zeroes is considered
-  // invalid. Can be set for logs that are part of a particular processing span.
-  // If span_id is present trace_id SHOULD be also present. [Optional].
+  // invalid.
+  //
+  // This field is optional. If valid span_id is present then trace_id SHOULD be also present.
```

Comment:
```suggestion
  // This field is optional. If a producer supplies a valid span_id then trace_id SHOULD be also present.
```
(because as written the phrase is meaningless to a consumer)

Reply:
Reworded, PTAL.

Reply:
Reworded, PTAL.